cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -192,6 +195,52 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
       optionalResultExplanation = "Note that it can only be created once and that the index name must be unique.",
       assertions = _ => assertTokenLookupIndexExists("rel_type_lookup_index", isNodeIndex = false)
     )
+    prepareAndTestQuery(
+      title = "Create a token lookup index specifying the index provider",
+      text = "Token lookup indexes allow setting the index provider using the `OPTIONS` clause. " +
+        "Note that the index is not immediately available, but is created in the background.",
+      prepare = _ => executePreparationQueries(List("DROP INDEX node_label_lookup_index IF EXISTS")),
+      queryText = s"CREATE LOOKUP INDEX node_label_lookup_index_2 FOR (n) ON EACH labels(n) OPTIONS {indexProvider: '${TokenIndexProvider.DESCRIPTOR.name()}'}",
+      optionalResultExplanation = "Currently only one provider is available for token lookup indexes.",
+      assertions = _ => assertTokenLookupIndexExists("node_label_lookup_index_2", isNodeIndex = true)
+    )
+  }
+
+  @Test def create_text_index() {
```

Comment:
I don't have any tests trying to use the index (not for lookup either I noticed), those only exists for btree. Do we want that kind of test?

Also, I don't know which of the indexes planner team have time to pick up and use in plans, but do we need to mention that somewhere in the docs?

Reply:
Using lookup is something that the planner team should have added in 4.3 I think, perhaps check with them if it is an oversight or if it is added somewhere else, like in the Query tuning chapter. 

I talked a bit with @tobias-johansson about indexes this morning and it seems a bit unclear if we will have planning support or not. I guess it would be worth mentioning somewhere that the indexes can only be created and not used for now, but not sure where it fits. Maybe we need to talk a bit about it with the people in the index channel and PM to give the correct message?

Reply:
planner team have been notified about the missing lookup tests and that index usage tests are not done for text either

Answer: We don't document anything until supported by planner